### ISSUE TITLE
setting the height property causes the horizontal scroll to unnecessarily appear

### ISSUE THREAD

**Issue by bchr02** [openpsa/jsgrid#98]
Setting the "height" property causes the horizontal scroll to unnecessarily appear when the vertical bar is needed. (This happens in Chrome but not in IE. Haven't tested it in any other browser.)

Even if the "width" property is defined and is more than wide enough to handle the table, the issue still occurs.

Example shown here: https://www.tesllc.aero/temp/jqgrid/

The only way that I have been able to temporarily fix this is by adding the following css:
````css
.ui-jqgrid .ui-jqgrid-bdiv {  overflow-x:hidden; }
````

**Comment by bouks** [comment]
Hi.

I don't see this bug in your example. Could you tell us your configuration ?
- navigator and version
- MacOs, Windows, Linux

**Comment by flack** [comment]
Tested with Chrome and Firefox under Linux, but I don't see a horizontal scrollbar either:

![chrome](https://cloud.githubusercontent.com/assets/425166/5979313/014aec6e-a8a7-11e4-820c-9fff7318bf88.png)

BTW: I've made some changes to the border rendering recently (see #88). theoretically, they should not have any effect on your issue, but who knows. You could try updating the CSS and see if it changes anything

**Comment by bchr02** [comment]
I am using Chrome version 40 (most recent) on Windows 7.

After seeing your replies, I tried it on a similar computer and was also not able to replicate it. Then I remembered that I have Windows Custom Font Scaling set to 110%. I bet that is what is causing it. Unfortunately changing it back to 100% requires a log off and I can't do that right now. 

I guess we can chalk it up to a Chrome bug. This is the first time having that setting set to 110% caused a web page to render differently. It normally would only help with Windows GUI fonts...
